samples/sample-bugsnag/CrashKiOSSampleIOS/CrashKiOSSampleIOS/AppDelegate.swift

```diff
@@ -18,6 +18,7 @@ import Bugsnag
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        Bugsnag.start()
```

Comment:
This should be called from within BugsnagConfigKt.startBugsnag(config: config)

Reply:
ok removed